### PR TITLE
Completed react-function-components exercise. - Daniel Cho

### PR DESCRIPTION
**Pull request by Daniel-J-Cho**
Screen Shot 2022-05-18 at 12 56 13 PM

### CODE REVIEW

File: react-function-components/src/index.jsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+
+function CustomButton(props) {
+  return <button>Click Me!</button>;
+}
+
+const root = ReactDOM.createRoot(document.getElementById('root'));
+const element = CustomButton();
+root.render(element);
```
Comment (scbowler) [inline react-function-components/src/index.jsx]: We don't call components like regular functions, we use them as JSX:

```JSX
root.render(<CustomButton />);
```

**Review by scbowler — CHANGES_REQUESTED** [review]
Heading in the right direction, but an update is needed. Let me know if you have any questions.

File: react-function-components/src/index.jsx (hunk 1 above)
Comment (Daniel-J-Cho) [reply react-function-components/src/index.jsx]: Oh ok, I'll go ahead and fix that.  Thank you

Comment (Daniel-J-Cho) [reply react-function-components/src/index.jsx]: Screen Shot 2022-05-18 at 1 04 45 PM

**Review by scbowler — APPROVED** [review]
Great job implementing the feedback, looks good! 👍

**Pull request merged**